source/kraken/tests/data_manager_with_type_data_test.cpp

```diff
@@ -0,0 +1,121 @@
+/* Copyright © 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+
+This file is part of Navitia,
+    the software to build cool stuff with public transport.
+
+Hope you'll enjoy and contribute to this project,
+    powered by Canal TP (www.canaltp.fr).
+Help us simplify mobility and open public transport:
+    a non ending quest to the responsive locomotion way of traveling!
+
+LICENCE: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Stay tuned using
+twitter @navitia
+IRC #navitia on freenode
+https://groups.google.com/d/forum/navitia
+www.navitia.io
+*/
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE data_manager_test
+#include <boost/test/unit_test.hpp>
+
+#include "kraken/data_manager.h"
+#include "type/data.h"
+
+static const std::string fake_data_file = "fake_data.nav.lz4";
+static const std::string fake_disruption_path = "fake_disruption_path";
+
+// We create a empty data with lz4 format in current directory.
+#define CREATE_FAKE_DATA(fake_file_name) \
+    navitia::type::Data data(0);         \
+    data.save(fake_data_file);           \
```

Comment:
Can't we use tmp directory instead?
Not fan of tests creating files... Or we have to clean after.